### DISCUSSION TITLE
Volume not meshing correctly

### DISCUSSION THREAD

**Discussion by James-houghton14** [GeoChemFoam/GeoChemFoam#47] [Q&A] [answered]
Hi All,

I hope that you might be able to help. I have successfully run the workflow from tutorial 1 on my own CT images. However, when I tried to rerun a sample today I noticed that starting runSnappyHexMesh.sh was using about half the normal amount of RAM and finished very rapidly. When I try to load the mesh in Paraview I get the error message "Paraview_Loader.foam contains no meshes". I have not changed any input setting from when it was working successfully a few days ago. 

I also tried to rerun the tutorial using the Ketton image, and I am experiencing the same error. 

Cheers,
James.

**Comment by GeoChemFoam** [comment, marked as answer]
HI James,

If it’s working with the tutorial images and not your own then it is likely something on your end. My advice is to start with a small and troubleshoot changing only one thing at a time. Go through the meshing script line by line and try and figure out where your image does not for the criteria set.

Hannah


From: James-houghton14 ***@***.***>
Date: Wednesday, 17 April 2024 at 13:06
To: GeoChemFoam/GeoChemFoam ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GeoChemFoam/GeoChemFoam] Volume not meshing correctly (Discussion #47)
****************************************************************
Caution: This email originated from a sender outside Heriot-Watt University.
Do not follow links or open attachments if you doubt the authenticity of the sender or the content.
****************************************************************


Hi All,

I hope that you might be able to help. I have successfully run the workflow from tutorial 1 on my own CT images. However, when I tried to rerun a sample today I noticed that starting runSnappyHexMesh.sh was using about half the normal amount of RAM and finished very rapidly. When I try to load the mesh in Paraview I get the error message "Paraview_Loader.foam contains no meshes". I have not changed any input setting from when it was working successfully a few days ago.

I also tried to rerun the tutorial using the Ketton image, and I am experiencing the same error.

Cheers,
James.

—
Reply to this email directly, view it on GitHub<https://github.com/GeoChemFoam/GeoChemFoam/discussions/47>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWDOAFV42Z6SMQBVRZBGQB3Y5ZQUVAVCNFSM6AAAAABGLFU2SKVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGUYTSMRRGM>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

________________________________

Founded in 1821, Heriot-Watt is a leader in ideas and solutions. With campuses and students across the entire globe we span the world, delivering innovation and educational excellence in business, engineering, design and the physical, social and life sciences. This email is generated from the Heriot-Watt University Group, which includes:

  1.  Heriot-Watt University, a Scottish charity registered under number SC000278
  2.  Heriot- Watt Services Limited (Oriam), Scotland's national performance centre for sport. Heriot-Watt Services Limited is a private limited company registered is Scotland with registered number SC271030 and registered office at Research & Enterprise Services Heriot-Watt University, Riccarton, Edinburgh, EH14 4AS.

The contents (including any attachments) are confidential. If you are not the intended recipient of this e-mail, any disclosure, copying, distribution or use of its contents is strictly prohibited, and you should please notify the sender immediately and then delete it (including any attachments) from your system.
